bin/plugin/commands/performance.js

```diff
@@ -337,8 +337,6 @@ async function runPerformanceTests( branches, options ) {
 							performanceTestDirectory,
 							'test/emptytheme'
 						),
-						'https://downloads.wordpress.org/theme/twentytwentyone.1.7.zip',
```

Comment:
I see that the bundled themes maintain these versions, thanks from that. We can deal with updates separately. When we update them, how do you recommend going about? Download the corresponding ZIP and update the contents here? It seems safer than taken them from GitHub.

Reply:
See https://github.com/WordPress/gutenberg/pull/50905#discussion_r1205364964